File: app/src/main/java/example/dgorod/widget/MyApp.java
package example.dgorod.widget;

import android.app.Application;

import timber.log.Timber;

/**
 * @author Dmytro Gorodnytskyi
 *         on 18-Apr-17.
 */

public class MyApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: app/src/main/java/example/dgorod/widget/MyApp.kt
package example.dgorod.widget

import android.app.Application
import timber.log.Timber

/**
 *
 * @author Dmytro Gorodnytskyi
 * on 22-May-17.
 */
class MyApp : Application() {

    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }
}